### PR TITLE
add support for other currencies than the default USD

### PR DESCRIPTION
**Pull request by TNGPS**
added new method to set your currency like this:
       
AnalyticsProvider.setCurrency('EUR');

should be used only in EC mode ..

### CODE REVIEW

File: src/angular-google-analytics.js [hunk 1]
```diff
@@ -275,6 +280,9 @@ angular.module('angular-google-analytics', [])
               $window.ga('require', 'ecommerce', 'ecommerce.js');
             } else {
               $window.ga('require', 'ec', 'ec.js');
+              if(curency != 'USD'){
```
Comment (TNGPS) [inline src/angular-google-analytics.js]: typo: should be "currency" not "curency" !

Comment (justinsa) [inline src/angular-google-analytics.js]: Should use !== and there should be a space on either side of the conditional, if (...) {.

Comment (justinsa) [inline src/angular-google-analytics.js]: Is there a reason to make this USD check in the first place? Is there anything wrong with always sending the &cu value even if it contains 'USD' as the currency?

**Comment by justinsa** [comment]
Even though this is fairly trivial in behavior it would be wise to add a few tests for this so future contributions can't accidentally modify the expected behavior.

Consider these tests to cover the functionality.

1. Confirm the default value is sent (or not sent) for EC.
2. Confirm a value other than USD is sent for EC.
3. Confirm the default value is not sent for non-EC.
4. Confirm a value other than USD is not sent for non-EC.

File: src/angular-google-analytics.js (hunk 1 above)
Comment (stramel) [inline src/angular-google-analytics.js]: I seem to agree with @justinsa, I don't see the value in this check.

**Comment by justinsa** [comment]
@TNGPS Are you going to update this PR with our feedback?

**Comment by justinsa** [comment]
Closing this abandoned PR as I have created a complimentary one that handles the feedback received on this one.

**Pull request closed without merging**